src/template.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Movie website app</title>
+
+  <!-- Roboto fonts -->
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Poppins:wght@300&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,700&display=swap"
+    rel="stylesheet" />
+  <!-- font awesome -->
+  <script src="https://kit.fontawesome.com/6e8f4ae605.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <header>
+    <nav class="nav">
+      <div class="logo-movie">
+        <a href="#" class="logo">
+          <span>D</span>
+          oc
+          <span>V</span>
+          ic
+        </a>
+        <span>movies</span>
+      </div>
```

Comment:
- [ ] [**OPTIONAL**] I noticed that your header does not have the number of all the movies , kindly ensure you add it to get the job done 👍 

![Capture](https://github.com/Dorcas126/JavaScript-capstone-project/assets/80458199/9e6f845c-9390-4a40-8bd1-26b838ed8bb8)


![Capture](https://github.com/Dorcas126/JavaScript-capstone-project/assets/80458199/5146e161-060d-45f9-9b6b-8963caffbde3)

Reply:
- [ ] [OPTIONAL] I think it will be cool you implement this kind suggestion for improvement by the reviewer :kneeling_person: What you have here is a counter for another project - spaceship, not for this movie web app.